.kube/app/Dockerfile

```diff
@@ -34,11 +34,7 @@ RUN apt-get install -y \
     s3cmd
 
 RUN docker-php-ext-configure gd --with-freetype --with-jpeg
-RUN docker-php-ext-install -j$(nproc) gd
-RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath bz2 zip soap
-RUN docker-php-ext-install opcache
-RUN docker-php-ext-configure intl
-RUN docker-php-ext-install intl
+RUN docker-php-ext-install -j$(nproc) gd pdo_mysql mbstring exif pcntl bcmath bz2 zip soap intl opcache
```

Comment:
Consolidated extensions to one line to speed up building container. Each time this command is run a build environment has to be created.